README.md

```diff
@@ -5,7 +5,7 @@
 > A simple package to map your routes for your expressjs application
 
 ---
-**IMPORTANT: v1.0.2 fixed a security vulnerability. Every version up to v1.0.1 is not save for production. Update your current version to v1.0.2. You can find more information [here](https://github.com/aichbauer/express-routes-mapper/issues/15).**
+**IMPORTANT: v1.0.2 fixed a security vulnerability. Every version up to v1.0.1 is not safe for production. Update your current version to v1.0.2. You can find more information [here](https://github.com/aichbauer/express-routes-mapper/issues/15).**
```

Comment:
Could you also change 

> v1.0.2

 to 

> v1.0.2 or higher

?

Reply:
Sure!